packages/@aws-cdk/aws-glue/lib/connection.ts

```diff
@@ -0,0 +1,114 @@
+import ec2 = require('@aws-cdk/aws-ec2');
```

Comment:
We switched the import style in the meantime :).

```ts
import * as ec2 from '@aws-cdk/aws-ec2';
```

Reply:
fixed